test/client/domains.js

```diff
@@ -51,12 +51,14 @@ describe(`domains test`, () => {
 
   it("should successfully match valid venmo domain", () => {
     const validDomains = [
-      "www.venmo.com",
-      "www.venmo.com:8000",
-      "account.qa.venmo.com",
-      "www.account.qa.venmo.com",
-      "venmo.com",
-      "id.venmo.com",
+      "https://venmo.com",
```

Comment:
can we update the negative cases too
`"should not match invalid venmo domains`

Reply:
Those are still good. It's just that we didn't expect these correctly. Needed to have protocol and port because that's what is required when checking for x-domain.

Reply:
I'll add some anyway